neural_compressor/torch/quantization/config.py

```diff
@@ -368,20 +369,15 @@ def register_supported_configs(cls) -> List[OperatorConfig]:
         cls.supported_configs = supported_configs
 
     @staticmethod
-    def get_model_info(model: torch.nn.Module) -> List[Tuple[str, Callable]]:
-        white_list = (torch.nn.Linear,)
-        filter_result = []
-        for op_name, module in model.named_modules():
-            if isinstance(module, white_list):
-                pair = (op_name, type(module).__name__)
-                filter_result.append(pair)
-        logger.debug(f"Get model info: {filter_result}")
-        return filter_result
+    def get_model_info(model: torch.nn.Module, example_inputs) -> List[Tuple[str, Callable]]:
+        from neural_compressor.torch.algorithms.static_quant import _get_quantizable_ops_recursively
```

Comment:
Please avoid importing components from the `algorithms` folder as this could result in a circular import.

Reply:
I suggest moving `_get_quantizable_ops_recursively` into utility.py if possible.
nit: avoid importing a function starting with `_`.